office/libreoffice/Portfile

```diff
@@ -155,15 +155,11 @@ if {${os.platform} eq "darwin" && ${os.major} < 19} {
     }
 }
 
-patchfiles \
-    patch-configure.diff \
-    patch-ucb_ucp_webdav_webdavprovider_cxx.diff \
-    patch-unpack-sources-fix-for-bsd-find.diff
-
 set product_name LibreOffice
 
 pre-configure {
     system "cd ${worksrcpath} && env NOCONFIGURE=1 ./autogen.sh"
+    reinplace "/# if the systemd-detect-virt command/d" Makefile.in
```

Comment:
I assume this reinplace is meant to be equivalent to this upstream commit?

https://github.com/LibreOffice/core/commit/cfab695d0e3bd5889cfa4b6e81300dd11d4eef72

Patchfiles are preferred over reinplaces. Can a patch be used instead? If Makefile.in is a generated file, can you patch the file it is generated from?

Reply:
Yes. I reported that issue on the mailing list. I will update this with that commit patch.